packages/shared/src/runtime/logger.ts

```diff
@@ -0,0 +1,3 @@
+import { Logger } from "@replay-cli/shared/logger/logger";
```

Comment:
If this tries to import from "../logger" the build fails:
```ts
import { Logger } from "../logger";
```

Maybe bc of the same file name? Seems like that should work (and it's what Code auto-completes)